_includes/components/indicator/metadata-tabs-default.html

```diff
@@ -1,9 +1,11 @@
 <li role="presentation" class="nav-item active">
   <a class="nav-link" data-toggle="tab" id="tab-national" href="#national" aria-controls="national" role="tab" {% include autotrack.html preset="tab_meta_national" category="Tab change" action="Change metadata view" label="Change to National Metadata tab" %}>{{ site.non_global_metadata | default: 'indicator.national_metadata' | t }}</a>
 </li>
+{% unless page.indicator.standalone %}
 <li role="presentation" class="nav-item">
   <a class="nav-link" data-toggle="tab" id="tab-global" href="#global" aria-controls="global" role="tab" {% include autotrack.html preset="tab_meta_global" category="Tab change" action="Change metadata view" label="Change to Global Metadata tab" %}>{{ page.t.indicator.global_metadata }}</a>
 </li>
+{% endunlesse %}
```

Comment:
There's a typo here which is causing an error

Reply:
Thanks @LucyGwilliamAdmin, just fixed that.